CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added `signInWithNewPassword` GraphQL API for signing in with a new password.
+  - The `signIn` GraphQL API now returns an error if the account was never signed
```

Comment:
I think we need to break the line before the word "signed". Could you check your IDE settings?

Reply:
I checked for modification using the cmd + q command in the VSCode IDE.
thank you.